libs/scully/src/lib/routerPlugins/handledRoute.interface.ts

```diff
@@ -17,7 +17,7 @@ export interface HandledRoute {
   /** the _complete_ route */
   route: string;
   /** String, must be an existing plugin name. mandatory */
-  type: string;
+  type?: string;
```

Comment:
Is there a default type in case this property is empty?

Reply:
Yes, it can't be empty, the type form the configfile will be inculded